eth2util/deposit/deposit.go

```diff
@@ -207,6 +174,8 @@ func networkToForkVersion(network string) eth2p0.Version {
 		return [4]byte{0x70, 0x00, 0x00, 0x69}
 	case "gnosis":
 		return [4]byte{0x00, 0x00, 0x00, 0x64}
+	case "mainnet": // Default to mainnet
+		fallthrough
 	default:
 		return [4]byte{0x00, 0x00, 0x00, 0x00}
```

Comment:
it isn't obviously that this is mainnet

Reply:
oh i get, this is the first usecase where `fallthrough` makes sense. thanks for pointing this!